tests/console/sshd.pm

```diff
@@ -64,7 +64,7 @@ sub run {
 
     # poo#80716 Test all available ciphers, key exchange algorithms, host key algorithms and mac algorithms.
     assert_script_run "echo 'sshd.pm: Testing cryptographic policies' | logger";
-    test_cryptographic_policies(remote_user => $ssh_testman);
+    test_cryptographic_policies(remote_user => $ssh_testman, system_role => get_var('SYSTEM_ROLE'));
```

Comment:
We don't use this setting in the original job and in general we try to not use settings as much as possible. why? this is a good example, is not really clear why a minimal role cannot upload logs without tracking to your pr, only reading the code would be insufficient.
Instead I would recommend that you check directly if curl is there to upload them or not.